govcd/vm_groups.go

```diff
@@ -0,0 +1,143 @@
+package govcd
+
+import (
+	"fmt"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"net/url"
+)
+
+// LogicalVmGroup is used to create VM Placement Policies.
+type LogicalVmGroup struct {
+	LogicalVmGroup *types.LogicalVmGroup
+	client         *Client
+}
+
+// VmGroup is used to create VM Placement Policies.
+type VmGroup struct {
+	VmGroup *types.QueryResultVmGroupsRecordType
```

Comment:
I couldn't find a specific API endpoint with an associated XML resource for this so I used the query type here.